e2e/features/reissue-document.feature

```diff
@@ -0,0 +1,22 @@
+@reissue-document @migrated-to-exui 
+Feature: Reissue document
+
+    Scenario: Reissue an document to appellants
+        Given I presetup an "PIP" SYA case
+        And I am signed in as a Case Officer
+        And I navigate to an existing case
+        When I choose "Update to case data"
+        Then I should update case with a valid nino
+        When I switch to be a Judge
+        And I navigate to an existing case
+        And I choose "Issue directions notice"
+        And I allow the appeal to proceed
+        Then I should see Addition details in documents tab
+        When I switch to be a Case Officer
+        And I wait for Judge to sign out
```

Comment:
we don't need this step as we directly go the case url once after login https://github.com/hmcts/sscs-ccd-e2e-tests/blob/master/e2e/features/step_definitions/authentication.steps.ts#L39